boards/shields/x_nucleo_iks01a3/doc/index.rst

```diff
@@ -82,9 +82,9 @@ Programming
 
 Two samples are provided as examples for ``x-nucleo-iks01a3`` shield:
 
-- :ref:`x-nucleo-iks01a3-std-sample` application, to be used when the shield is configured
+- :zephyr:code-sample:`x-nucleo-iks01a3-std` sample application, to be used when the shield is configured
   in Standard Mode
-- :ref:`x-nucleo-iks01a3-shub-sample` application, to be used when the shield is configured
+- :zephyr:code-sample:`x-nucleo-iks01a3-shub` sample application, to be used when the shield is configured
```

Comment:
nit: Could be reformatted as coding style line length also applies to documentation sources.

Reply:
yes but coding style line length is 100 characters :)

Reply:
And this line is 106.

Reply:
> And this line is 106.

touché :) was looking at the wrong diff. It's on my list to find a good RST linter and hook it up to CI...